faults.csv

```diff
@@ -1,63 +1,70 @@
+MNT,Maintenance,Maintenance: Cavity is known to be bad and is actively being worked on,No further action required,CAV,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=10,SELCHAR=3",,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,HWMODE,,1,4,Maintenance,
+MND,Maintenance Done,Maintenance done but cavity has not been run up for validation,"If program permits, set online and verify stability. If stable, leave online. If unstable, comment on/reopen the relevant CATER and set offline",CAV,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=10,SELCHAR=3",,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,HWMODE,,3,4,Maintenance Done,
+RDY,Ready,Cavity usable but turned off for now,"If program permits, set online",CAV,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=10,SELCHAR=3",,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,HWMODE,,4,4,Ready,
+BSO,BSOIC Tripped Chain A,BSOIC tripped,Communicate the fault to the EOIC and await resolution,ALL,EDM,$EDM/pps/pps_sysw.edl,,,BSOC:SYSW:2:,SumyA,1,,2,BSOIC Tripped,
+BSO,BSOIC Tripped Chain B,BSOIC tripped,Communicate the fault to the EOIC and await resolution,ALL,EDM,$EDM/pps/pps_sysw.edl,,,BSOC:SYSW:2:,SumyB,1,,2,BSOIC Tripped,
+PPS,Not Ready for Beam Chain A,PPS: West Global not in Ready for Beam,Communicate the fault to the EOIC and await resolution,ALL,EDM,$EDM/pps/pps_linacwest.edl,,,PPS:SYSW:1:,BeamReadyA,1,,2,Not Ready for Beam,
+PPS,Not Ready for Beam Chain B,PPS: West Global not in Ready for Beam,Communicate the fault to the EOIC and await resolution,ALL,EDM,$EDM/pps/pps_linacwest.edl,,,PPS:SYSW:1:,BeamReadyB,1,,2,Not Ready for Beam,
+HWI,Hardware Initialization needed,Hardware Initialization (a.k.a. rack checkout) needs to be run or other LLRF chassis problem. Don't trust current values.,Verify that the rack is not actively being worked on and run Rack Hardware Init and Test > Rack Checkout > Go,RACK,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=8,SELCHAR=3",A,ACCL:{LINAC}:{CRYOMODULE}00:RACK{RACK}:,HWINITSUM,,2,2,,
+HWI,Hardware Initialization needed,Hardware Initialization (a.k.a rack checkout) needs to be run or other LLRF chassis problem. Don't trust current values.,Verify that the rack is not actively being worked on and run Rack Hardware Init and Test > Rack Checkout > Go,RACK,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=8,SELCHAR=3",B,ACCL:{LINAC}:{CRYOMODULE}00:RACK{RACK}:,HWINITSUM,,2,2,,
+HWR,Hardware Initialization running,Hardware Initialization (ak.a. rack checkout) currently running. Don't try to use system until this is complete,Wait for fault to clear,RACK,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=8,SELCHAR=3",A,ACCL:{LINAC}:{CRYOMODULE}00:RACK{RACK}:,HWINITSUM,,1,1,,
+HWR,Hardware Initialization running,Hardware Initialization (a.k.a. rack checkout) currently running. Don't try to use system until this is complete,Wait for fault to clear,RACK,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=8,SELCHAR=3",B,ACCL:{LINAC}:{CRYOMODULE}00:RACK{RACK}:,HWINITSUM,,1,1,Hardware Initialization needed or running,

```

Comment:
Can we edit the text in the final column to just say "Hardware Initialization running"?